### DISCUSSION TITLE
New Release?

### DISCUSSION THREAD

**Discussion by amasses** [Versent/saml2aws#1118] [General]
Hi @mapkon 

I just wanted to reach out to see if it would be possible for a new version to be released give there have been a number of changed merged in since the last one in July.

Thanks for your consideration in this matter!

**Comment by mapkon** [comment]
@amasses - thank you for the message. Yes, we can release. However, you would need to coordinate testing for release readiness. What that means is that we should review the delta and see if there are no regressions.

Once we confirm that the changes are ok and backward compatible (Semantically at least), then I am happy to release

Could you do that?

**Reply by amasses** [reply]
@mapkon in particular what are you looking for from me? 

I can test the Okta functionality we are waiting on (the number challenge changes recently merged in #1116) but anything else I can't really do because I don't use anything aside from Okta.

**Reply by mapkon** [reply]
Yeah, that will suffice @gliptak

**Comment by tinaboyce** [comment]
Hi @amasses , huge apologies for the delay but I here to announce there is a new release v2.36.13 https://github.com/Versent/saml2aws/discussions/1176
